content/technical_challenges.tex

```diff
@@ -29,8 +29,8 @@ \subsection{7 vs. 7}
             \end{figure}
             Blue and red jerseys with numbers from 1 to 7 are provided by the OC/TC. If a team decides to play with own jerseys, they have to provide an additional one with number 7 of the same shape. 
 
-        \paragraph{WiFi}
-            The available Wifi package budget defined in \cref{sec:wireless} will be increase by \qty{40}{\percent} both for the initial budget and for the budget for playing time extensions.  
+        \paragraph{Wireless network}
+            The available wireless network package budget defined in \cref{sec:wireless} will be increase by \qty{40}{\percent} both for the initial budget and for the budget for playing time extensions.  
```

Comment:
Should be not in this PR? Instead a commit on the rules themselves?

Reply:
Yeah, should have been, but I thought this PR has to be merged anyhow so I let it in.